lib/params.js

```diff
@@ -119,7 +106,23 @@ function params (options) {
       params.indentWidth = getIndentationFromStylelintRules(rules)
       params.stylelint = rules
     }
-    return loadStylelintExtendConfig(params, stylelint.config, cwd)
+    return loadStylelintExtendConfig(params, stylelint.config, options.configBasedir || cwd)
+  })
+}
+
+function loadStylelintConfig (options) {
+  // same as the options in stylelint. see: http://stylelint.io/user-guide/node-api/#options
+  try {
+    return require('stylelint/dist/buildConfig').default(options).catch(function() {
```

Comment:
There is a very similar code here:
https://github.com/spalger/gulp-jshint/blob/master/src/lint.js#L4